src/newSeed/baseStage.scss

```diff
@@ -476,6 +476,7 @@
     .infoContainer {
       & textarea {
         height: 7em;
+        padding: 12px;
```

Comment:
this looks good, but why only for this one page?  can you please make this apply to all the textareas in the app, not just this one page?  That means applying the padding in styles.scss instead of here.

Reply:
Fixed that in the styles.scss (and removed the declaration from baseStage.scss).